tools/public_api_guard/material/list.md

```diff
@@ -179,18 +179,19 @@ export class MatListOption extends MatListItemBase implements ListOption, OnInit
     _titles: QueryList<MatListItemTitle>;
     toggle(): void;
     _toggleOnInteraction(): void;
+    togglePosition: MatListOptionTogglePosition;
     // (undocumented)
     _unscopedContent: ElementRef<HTMLSpanElement>;
     get value(): any;
     set value(newValue: any);
     // (undocumented)
-    static ɵcmp: i0.ɵɵComponentDeclaration<MatListOption, "mat-list-option", ["matListOption"], { "checkboxPosition": "checkboxPosition"; "color": "color"; "value": "value"; "selected": "selected"; }, { "selectedChange": "selectedChange"; }, ["_lines", "_titles"], ["[matListItemAvatar],[matListItemIcon]", "[matListItemTitle]", "[matListItemLine]", "*", "mat-divider"], false, never>;
+    static ɵcmp: i0.ɵɵComponentDeclaration<MatListOption, "mat-list-option", ["matListOption"], { "togglePosition": "togglePosition"; "color": "color"; "value": "value"; "selected": "selected"; }, { "selectedChange": "selectedChange"; }, ["_lines", "_titles"], ["[matListItemAvatar],[matListItemIcon]", "[matListItemTitle]", "[matListItemLine]", "*", "mat-divider"], false, never>;
     // (undocumented)
     static ɵfac: i0.ɵɵFactoryDeclaration<MatListOption, [null, null, null, null, null, { optional: true; }, { optional: true; }]>;
 }
 
 // @public
-export type MatListOptionCheckboxPosition = 'before' | 'after';
+export type MatListOptionTogglePosition = 'before' | 'after';
```

Comment:
This is a breaking change. I know that another comment said that it's a new component, but my understanding is that we still can't make changes like this in an RC, it would've had to be done before the feature freeze.

Reply:
I guess we should get clarification on it, but to me it makes more sense to just change it than release a new component with an already deprecated API

Reply:
Yeah, technically once we leave feature freeze breaking changes are not allowed. I remember framework had an exemption too for forms or the image directive IIRC?! so yes, this might be okay to be skipped.

Reply:
Okay, I made it so `MatListOptionCheckboxPosition` is deprecated and changed the target of this PR to minor.